pokemon-app/src/App.tsx

```diff
@@ -1,12 +1,69 @@
-import React from 'react';
-import './App.css';
+import React, { useEffect, useState } from 'react';
+import './styles/mainStyle.scss'
+import './styles/footer.scss'
+import Header from './components/Header/Header';
+import Footer from './components/Footer/Footer';
+import Gallery from './components/Gallery/Gallery';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import PokemonPage from './components/PokemonPage/PokemonPage';
+import { connect, useDispatch } from 'react-redux';
+import axios from 'axios';
+import { Pokemon } from './types';
+import { addPokemonArr } from './actions';
 
 function App() {
+  const [pokemons, setPokemons]: any[] = useState([])
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+      const limit = 100
```

Comment:
Replace these constants out of the component and name as:

const LIMIT = 100 as const; // Read about what 'as const' does mean.